## Table legends

Prioritization results of network genes are summarized as two tables,
one for network transcription factors (TFs)
and the other for network target genes (TGs).

- `P1`: posterior probability that at least one SNP within
100 kb of the transcribed region of a given network gene
(specified by `Gene`, `Chr.`, `Start` and `End` columns)
has non-zero effect on the trait of interest;

- `H-distance`: the physical distance, in base pair,
between a given network gene and its nearest GWAS hit;

- `Nearest hit`: the nearest GWAS hit to a given network gene,
reported in the corresponding GWAS publication.
The "none" values in this column indicate that there is no
published GWAS hit on the same chromosome as the network gene.

Note that `Baseline P1` and `Enrichment P1` are obtained from
fitting RSS-NET baseline and enrichment models respectively.
Differences between `Baseline P1` and `Enrichment P1` reflect
the influence of network enrichment on genetic associations,
which can help identify putatively new trait-associated genes. 

```{r, include=FALSE}
# load packages
suppressPackageStartupMessages(library(dplyr))
suppressPackageStartupMessages(library(DT))

# load RSS-NET prioritization results
res_df <- readRDS(paste0(proj_path,res_dir,gwas_name,"/",gwas_name,"_gene.rds"))

# annotate each gene by the nearest GWAS hit
pip.df <- unique(res_df[,c("gene_nid","chr","start","stop")])
names(pip.df) <- paste0("seg.",c("id","chr","start","end"))

source("~/Dropbox/rss/Rcode/vp_util.R")

pip.df <- add.nearhits(pip.df, hit.df)

# add GWAS hit annotations to results
joinls <- c("chr"="seg.chr","start"="seg.start","stop"="seg.end","gene_nid"="seg.id")
res_df <- dplyr::inner_join(res_df,pip.df,by=joinls)
rm(hit.df,pip.df,joinls)

# tidy up results for display
res_df$p1_diff <- res_df$p1 - res_df$base_p1

tf_df <- dplyr::arrange(res_df[res_df$annot=="TF", ], -p1_diff, -p1, -hit.distance)
tg_df <- dplyr::arrange(res_df[res_df$annot=="TG", ], -p1_diff, -p1, -hit.distance)

keep_cols <- c("hgnc_symbol","chr","start","stop","base_p1","p1","net","hit.distance","nearest.hit")
show_cols <- c("Gene","Chr.","Start","End","Baseline P1","Enrichment P1","Network","H-distance","Nearest hit")

tf_df <- tf_df[,keep_cols]
tg_df <- tg_df[,keep_cols]

names(tf_df) = names(tg_df) = show_cols

tf_dt <- DT::datatable(tf_df, rownames=FALSE, class="white-space: nowrap", escape=FALSE)
tf_dt <- tf_dt %>% DT::formatRound(c("Baseline P1","Enrichment P1"), 4)

tg_dt <- DT::datatable(tg_df, rownames=FALSE, class="white-space: nowrap", escape=FALSE)
tg_dt <- tg_dt %>% DT::formatRound(c("Baseline P1","Enrichment P1"), 4)
```

## Results of TFs

```{r, echo=FALSE}
tf_dt
```

## Results of TGs

```{r, echo=FALSE, warning=FALSE}
tg_dt
```


