<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2019-08-19" />

<title>Cross-reference results of Alzheimer’s disease</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-net-results</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="https://suwonglab.github.io/rss-net/results.html">Home</a>
</li>
<li>
  <a href="https://stephenslab.github.io/rss/news.html">News</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://suwonglab.github.io/rss-net/results.html">
    <span class="fab fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Cross-reference results of Alzheimer’s
disease</h1>
<h4 class="author">Xiang Zhu</h4>
<h4 class="date">2019-08-19</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rss-net-results/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20180719code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20180719)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20180719code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20180719)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssnetresultstreeb0dad18445c743649619896dc5bec8adfe7703d9targetblankb0dad18a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/rss-net-results/tree/b0dad18445c743649619896dc5bec8adfe7703d9" target="_blank">b0dad18</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssnetresultstreeb0dad18445c743649619896dc5bec8adfe7703d9targetblankb0dad18a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/rss-net-results/tree/b0dad18445c743649619896dc5bec8adfe7703d9" target="_blank">b0dad18</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/load2013_xref.Rmd</code>) and
HTML (<code>docs/load2013_xref.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/57af77473122877ed7ac45d7fbc45065500a1332/docs/load2013_xref.html" target="_blank">57af774</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/f669e3e251d1c0f5c2b2cdeca5cf32f778e1ceda/docs/load2013_xref.html" target="_blank">f669e3e</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/f40af1ccbbe02b956b9ec6066b3ccede2b722aea/docs/load2013_xref.html" target="_blank">f40af1c</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-11-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/04e60bf5b96a96203fac26147447bec9a8a7a214/docs/load2013_xref.html" target="_blank">04e60bf</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-03-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/8fde153b2847f7e0e68a7dc4e00e36f9afb6380b/docs/load2013_xref.html" target="_blank">8fde153</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-02-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/8209433cdfb26fc8034e818434b5ab819a2ba7e9/docs/load2013_xref.html" target="_blank">8209433</a>
</td>
<td>
xiangzhu
</td>
<td>
2020-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/d0646877c850c033f276df70cdd0e4f19d236384/docs/load2013_xref.html" target="_blank">d064687</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/ef65cf216148492cc8a82d299ccd812cba540d06/docs/load2013_xref.html" target="_blank">ef65cf2</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-net-results/blob/f71f993b3edb469c5100b240e25e03f45f4c821f/analysis/load2013_xref.Rmd" target="_blank">f71f993</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-06
</td>
<td>
wflow_publish("analysis/load2013_xref.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/8677943e477065617b778d95abb5d669c97507ec/docs/load2013_xref.html" target="_blank">8677943</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/115fb0949f37a887d72951f5966cc27a1030ff58/docs/load2013_xref.html" target="_blank">115fb09</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/9173a0c975a0ce79bc9c304e8ea67acadbd08e16/docs/load2013_xref.html" target="_blank">9173a0c</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/985be58dba9126c32aed57887d92a161aad8bdf3/docs/load2013_xref.html" target="_blank">985be58</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/6615e479947d0dfd9ca78ff9979d4acfb441eb35/docs/load2013_xref.html" target="_blank">6615e47</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/bad6f44923447fe1df7ba0d55c53b9d5d709f8ce/docs/load2013_xref.html" target="_blank">bad6f44</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/8ef19111a4e636b95ebe12300cf22217f5c45a0d/docs/load2013_xref.html" target="_blank">8ef1911</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-12-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-net-results/3238535b7fb8f16a713020beca77434044e4a52d/docs/load2013_xref.html" target="_blank">3238535</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-11-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-net-results/blob/ab62d6ae0048dc7e0492f40095ed1a6f321a2c80/analysis/load2013_xref.Rmd" target="_blank">ab62d6a</a>
</td>
<td>
xiangzhu
</td>
<td>
2019-11-30
</td>
<td>
wflow_publish("analysis/load2013_xref.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="therapeutic-target-database-ttd" class="section level2">
<h2>Therapeutic Target Database (TTD)</h2>
<p><br></p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-df4a1fd35b70caa27ccd" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-df4a1fd35b70caa27ccd">{"x":{"filter":"none","vertical":false,"data":[["CLU","CLU","CLU","EPHX2","EPHX2","APOE","APOE","APOE","APOE","CR2","CR1","CASP2","CASP2","CASP2","TTR","TTR","TTR","TTR","TTR","PTK2B","CD33","CD33","CD33","CD33","CD33","CD33","CD33"],["TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG"],[0.9999999999976262,0.9999999999976262,0.9999999999976262,0.9999999999976743,0.9999999999976743,1,1,1,1,0.9999992673703606,0.9999992674241207,0.9881764318168151,0.9881764318168151,0.9881764318168151,0.6378216689037693,0.6378216689037693,0.6378216689037693,0.6378216689037693,0.6378216689037693,0.086992926986042,0.2248577643798382,0.2248577643798382,0.2248577643798382,0.2248577643798382,0.2248577643798382,0.2248577643798382,0.2248577643798382],[1,1,1,1,1,1,1,1,1,0.9999999014520241,0.9999999015094846,0.9979312509452082,0.9979312509452082,0.9979312509452082,0.9211139741525487,0.9211139741525487,0.9211139741525487,0.9211139741525487,0.9211139741525487,0.4208427963442954,0.5754852943811206,0.5754852943811206,0.5754852943811206,0.5754852943811206,0.5754852943811206,0.5754852943811206,0.5754852943811206],[1,1,1,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999750058211,0.9999999409479006,0.9994992868035154,0.9994992868035154,0.9994992868035154,0.9356918128525192,0.9356918128525192,0.9356918128525192,0.9356918128525192,0.9356918128525192,0.7544708751485025,0.6940905161968751,0.6940905161968751,0.6940905161968751,0.6940905161968751,0.6940905161968751,0.6940905161968751,0.6940905161968751],["T95899","T95899","T95899","T35734","T35734","T21689","T21689","T21689","T21689","T18059","T48457","T81103","T81103","T81103","T23389","T23389","T86462","T86462","T86462","T07087","T38950","T38950","T38950","T38950","T38950","T38950","T38950"],["Lung cancer [ICD-11: 2C25.0]","Breast cancer [ICD-11: 2C60-2C65]","Prostate cancer [ICD-11: 2C82.0]","Hypertension [ICD-11: BA00-BA04]","Chronic obstructive pulmonary disease [ICD-11: CA22]","Familial hypercholesterolemia [ICD-11: 5C80.00]","Nontraumatic intracerebral hemorrhage [ICD-11: 8B00.4]","Hyperlipidaemia [ICD-11: 5C80]","Glaucoma/ocular hypertension [ICD-11: 9C61]","Autoimmune diabetes [ICD-11: 5A10]","Macular degeneration [ICD-11: 9B78.3]","Ischemic optic neuropathy [ICD-11: 9C40.4]","Non-arteritic anterior ischemic optic neuropathy [ICD-11: 9C40.40]","Glaucoma/ocular hypertension [ICD-11: 9C61]","Amyloidosis [ICD-11: 5D00]","Amyloid cardiomyopathy [ICD-11: 5D00.20]","Hereditary amyloidosis [ICD-11: 5D00.2]","Amyloidosis [ICD-11: 5D00]","Cardiomyopathy [ICD-11: BC43]","Osteoporosis [ICD-11: FB83.0]","Acute myeloid leukaemia [ICD-11: 2A60]","Myelodysplastic syndrome [ICD-11: 2A37]","Acute myelogenous leukaemia [ICD-11: 2A41]","Myeloid leukaemia [ICD-11: 2B33.1]","leukaemia [ICD-11: 2A60-2B33]","Multiple myeloma [ICD-11: 2A83]","Haematopoietic/lymphoid cancer [ICD-11: 2B33.5]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Gene<\/th>\n      <th>Role<\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>base<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>near<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>bma<\/sup><\/th>\n      <th>TTD ID<\/th>\n      <th>Disease<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4]},{"name":"Gene","targets":0},{"name":"Role","targets":1},{"name":"<i>P<\/i><sub>1<\/sub><sup>base<\/sup>","targets":2},{"name":"<i>P<\/i><sub>1<\/sub><sup>near<\/sup>","targets":3},{"name":"<i>P<\/i><sub>1<\/sub><sup>bma<\/sup>","targets":4},{"name":"TTD ID","targets":5},{"name":"Disease","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
</div>
<div id="mouse-genome-informatics-mgi" class="section level2">
<h2>Mouse Genome Informatics (MGI)</h2>
<p><br></p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-e948555e45e1e9771a0d" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e948555e45e1e9771a0d">{"x":{"filter":"none","vertical":false,"data":[["BCL3","BCL3","BCL3","BCL3","BCL3","BCL3","BCL3","BIN1","BIN1","BIN1","BIN1","BIN1","BIN1","BIN1","BIN1","BIN1","BIN1","TOMM40","TOMM40","TOMM40","CLU","CLU","CLU","CLU","CLU","CLU","BCAM","BCAM","BCAM","BCAM","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","RELB","EPHX2","EPHX2","EPHX2","APOC1","APOC1","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOC2","MARK4","MARK4","MARK4","MARK4","TRAPPC6A","TRAPPC6A","TRAPPC6A","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","BLOC1S3","CEACAM16","CEACAM16","CEACAM16","CEACAM16","CR2","CR2","CR2","CR2","SORL1","SORL1","SORL1","SORL1","SORL1","SORL1","SORL1","PICALM","PICALM","PICALM","PICALM","PICALM","PICALM","PICALM","PICALM","PICALM","PICALM","EPHA1","EPHA1","EPHA1","EPHA1","EPHA1","ZYX","CASP2","CASP2","CASP2","CASP2","CASP2","CASP2","CLCN1","CLCN1","CLCN1","CLCN1","CLCN1","CLCN1","CLCN1","CLCN1","CLCN1","CD2AP","CD2AP","CD2AP","CD2AP","CD2AP","CD2AP","CD2AP","CD2AP","CD2AP","TTR","TTR","DSG2","DSG2","DSG2","DSG2","DSG2","DSG2","DSG2","RIN3","PTK2B","PTK2B","PTK2B","PTK2B","PTK2B","PTK2B","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","SLC24A4","CD33","CD33","CD33"],["TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG"],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999976262,0.9999999999976262,0.9999999999976262,0.9999999999976262,0.9999999999976262,0.9999999999976262,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999976743,0.9999999999976743,0.9999999999976743,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999986294,0.9999999999986294,0.9999999999986294,0.9999999999986294,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.9999999999972153,0.999999999873957,0.999999999873957,0.999999999873957,0.999999999873957,0.9999992673703606,0.9999992673703606,0.9999992673703606,0.9999992673703606,0.9982827944180628,0.9982827944180628,0.9982827944180628,0.9982827944180628,0.9982827944180628,0.9982827944180628,0.9982827944180628,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9968214201251187,0.9881846957817182,0.9881846957817182,0.9881846957817182,0.9881846957817182,0.9881846957817182,0.9881846626304907,0.9881764318168151,0.9881764318168151,0.9881764318168151,0.9881764318168151,0.9881764318168151,0.9881764318168151,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.7003741638158492,0.6378216689037693,0.6378216689037693,0.6378279987931024,0.6378279987931024,0.6378279987931024,0.6378279987931024,0.6378279987931024,0.6378279987931024,0.6378279987931024,0.09478988015113718,0.086992926986042,0.086992926986042,0.086992926986042,0.086992926986042,0.086992926986042,0.086992926986042,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.09491209568051062,0.2248577643798382,0.2248577643798382,0.2248577643798382],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999999923,0.9999999999999923,0.9999999999999923,0.9999999999999923,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999999487,0.9999999999950631,0.9999999999950631,0.9999999999950631,0.9999999999950631,0.9999999014520241,0.9999999014520241,0.9999999014520241,0.9999999014520241,0.9998404345572768,0.9998404345572768,0.9998404345572768,0.9998404345572768,0.9998404345572768,0.9998404345572768,0.9998404345572768,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.999651259191367,0.9979391139024153,0.9979391139024153,0.9979391139024153,0.9979391139024153,0.9979391139024153,0.9979390815168873,0.9979312509452082,0.9979312509452082,0.9979312509452082,0.9979312509452082,0.9979312509452082,0.9979312509452082,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9544074711743679,0.9211139741525487,0.9211139741525487,0.9211517461508155,0.9211517461508155,0.9211517461508155,0.9211517461508155,0.9211517461508155,0.9211517461508155,0.9211517461508155,0.4586478442063147,0.4208427963442954,0.4208427963442954,0.4208427963442954,0.4208427963442954,0.4208427963442954,0.4208427963442954,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.4589597959861796,0.5754852943811206,0.5754852943811206,0.5754852943811206],[1.000000000000001,1.000000000000001,1.000000000000001,1.000000000000001,1.000000000000001,1.000000000000001,1.000000000000001,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999999,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999997,0.9999999999999949,0.9999999999999949,0.9999999999999949,0.9999999999999949,0.9999999999999925,0.9999999999999925,0.9999999999999925,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999999617,0.9999999999989159,0.9999999999989159,0.9999999999989159,0.9999999999989159,0.9999999750058211,0.9999999750058211,0.9999999750058211,0.9999999750058211,0.9999365844083582,0.9999365844083582,0.9999365844083582,0.9999365844083582,0.9999365844083582,0.9999365844083582,0.9999365844083582,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9998720754149846,0.9995103875269091,0.9995103875269091,0.9995103875269091,0.9995103875269091,0.9995103875269091,0.9995077892221594,0.9994992868035154,0.9994992868035154,0.9994992868035154,0.9994992868035154,0.9994992868035154,0.9994992868035154,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9780467935464852,0.9356918128525192,0.9356918128525192,0.929205960903237,0.929205960903237,0.929205960903237,0.929205960903237,0.929205960903237,0.929205960903237,0.929205960903237,0.7748653433872378,0.7544708751485025,0.7544708751485025,0.7544708751485025,0.7544708751485025,0.7544708751485025,0.7544708751485025,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.7134973578122618,0.6940905161968751,0.6940905161968751,0.6940905161968751],["MGI:88140","MGI:88140","MGI:88140","MGI:88140","MGI:88140","MGI:88140","MGI:88140","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:108092","MGI:1858259","MGI:1858259","MGI:1858259","MGI:88423","MGI:88423","MGI:88423","MGI:88423","MGI:88423","MGI:88423","MGI:1929940","MGI:1929940","MGI:1929940","MGI:1929940","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:103289","MGI:99500","MGI:99500","MGI:99500","MGI:88053","MGI:88053","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88057","MGI:88054","MGI:1920955","MGI:1920955","MGI:1920955","MGI:1920955","MGI:1914341","MGI:1914341","MGI:1914341","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2678952","MGI:2685615","MGI:2685615","MGI:2685615","MGI:2685615","MGI:88489","MGI:88489","MGI:88489","MGI:88489","MGI:1202296","MGI:1202296","MGI:1202296","MGI:1202296","MGI:1202296","MGI:1202296","MGI:1202296","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:2385902","MGI:107381","MGI:107381","MGI:107381","MGI:107381","MGI:107381","MGI:103072","MGI:97295","MGI:97295","MGI:97295","MGI:97295","MGI:97295","MGI:97295","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:88417","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:1330281","MGI:98865","MGI:98865","MGI:1196466","MGI:1196466","MGI:1196466","MGI:1196466","MGI:1196466","MGI:1196466","MGI:1196466","MGI:2385708","MGI:104908","MGI:104908","MGI:104908","MGI:104908","MGI:104908","MGI:104908","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:2447362","MGI:99440","MGI:99440","MGI:99440"],["mortality/aging","hematopoietic system phenotype","immune system phenotype","craniofacial phenotype","growth/size/body region phenotype","cellular phenotype","homeostasis/metabolism phenotype","immune system phenotype","behavior/neurological phenotype","mortality/aging","reproductive system phenotype","cardiovascular system phenotype","endocrine/exocrine gland phenotype","neoplasm","muscle phenotype","homeostasis/metabolism phenotype","integument phenotype","mortality/aging","behavior/neurological phenotype","vision/eye phenotype","homeostasis/metabolism phenotype","muscle phenotype","behavior/neurological phenotype","immune system phenotype","renal/urinary system phenotype","cardiovascular system phenotype","renal/urinary system phenotype","digestive/alimentary phenotype","cardiovascular system phenotype","growth/size/body region phenotype","limbs/digits/tail phenotype","digestive/alimentary phenotype","renal/urinary system phenotype","immune system phenotype","respiratory system phenotype","liver/biliary system phenotype","behavior/neurological phenotype","reproductive system phenotype","mortality/aging","integument phenotype","hematopoietic system phenotype","cardiovascular system phenotype","growth/size/body region phenotype","muscle phenotype","cellular phenotype","homeostasis/metabolism phenotype","endocrine/exocrine gland phenotype","homeostasis/metabolism phenotype","cardiovascular system phenotype","vision/eye phenotype","liver/biliary system phenotype","homeostasis/metabolism phenotype","behavior/neurological phenotype","respiratory system phenotype","liver/biliary system phenotype","mortality/aging","hematopoietic system phenotype","cardiovascular system phenotype","hearing/vestibular/ear phenotype","nervous system phenotype","digestive/alimentary phenotype","vision/eye phenotype","immune system phenotype","homeostasis/metabolism phenotype","skeleton phenotype","renal/urinary system phenotype","integument phenotype","growth/size/body region phenotype","adipose tissue phenotype","endocrine/exocrine gland phenotype","taste/olfaction phenotype","muscle phenotype","cellular phenotype","homeostasis/metabolism phenotype","behavior/neurological phenotype","adipose tissue phenotype","homeostasis/metabolism phenotype","growth/size/body region phenotype","integument phenotype","pigmentation phenotype","vision/eye phenotype","craniofacial phenotype","cellular phenotype","growth/size/body region phenotype","integument phenotype","pigmentation phenotype","limbs/digits/tail phenotype","hearing/vestibular/ear phenotype","vision/eye phenotype","immune system phenotype","behavior/neurological phenotype","hematopoietic system phenotype","vision/eye phenotype","hearing/vestibular/ear phenotype","behavior/neurological phenotype","homeostasis/metabolism phenotype","homeostasis/metabolism phenotype","immune system phenotype","mortality/aging","hematopoietic system phenotype","nervous system phenotype","behavior/neurological phenotype","cardiovascular system phenotype","muscle phenotype","craniofacial phenotype","homeostasis/metabolism phenotype","growth/size/body region phenotype","immune system phenotype","pigmentation phenotype","mortality/aging","hematopoietic system phenotype","liver/biliary system phenotype","embryo phenotype","cellular phenotype","homeostasis/metabolism phenotype","integument phenotype","growth/size/body region phenotype","growth/size/body region phenotype","homeostasis/metabolism phenotype","limbs/digits/tail phenotype","reproductive system phenotype","immune system phenotype","cellular phenotype","nervous system phenotype","immune system phenotype","hematopoietic system phenotype","reproductive system phenotype","endocrine/exocrine gland phenotype","cellular phenotype","nervous system phenotype","skeleton phenotype","behavior/neurological phenotype","mortality/aging","integument phenotype","reproductive system phenotype","growth/size/body region phenotype","muscle phenotype","homeostasis/metabolism phenotype","cellular phenotype","homeostasis/metabolism phenotype","growth/size/body region phenotype","hematopoietic system phenotype","cardiovascular system phenotype","endocrine/exocrine gland phenotype","immune system phenotype","renal/urinary system phenotype","mortality/aging","vision/eye phenotype","homeostasis/metabolism phenotype","digestive/alimentary phenotype","immune system phenotype","mortality/aging","cardiovascular system phenotype","muscle phenotype","cellular phenotype","integument phenotype","homeostasis/metabolism phenotype","immune system phenotype","cardiovascular system phenotype","hematopoietic system phenotype","homeostasis/metabolism phenotype","cellular phenotype","muscle phenotype","growth/size/body region phenotype","taste/olfaction phenotype","endocrine/exocrine gland phenotype","craniofacial phenotype","homeostasis/metabolism phenotype","skeleton phenotype","hematopoietic system phenotype","mortality/aging","nervous system phenotype","immune system phenotype","homeostasis/metabolism phenotype","hematopoietic system phenotype","immune system phenotype"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Gene<\/th>\n      <th>Role<\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>base<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>near<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>bma<\/sup><\/th>\n      <th>MGI ID<\/th>\n      <th>High-level mammalian phenotype<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4]},{"name":"Gene","targets":0},{"name":"Role","targets":1},{"name":"<i>P<\/i><sub>1<\/sub><sup>base<\/sup>","targets":2},{"name":"<i>P<\/i><sub>1<\/sub><sup>near<\/sup>","targets":3},{"name":"<i>P<\/i><sub>1<\/sub><sup>bma<\/sup>","targets":4},{"name":"MGI ID","targets":5},{"name":"High-level mammalian phenotype","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
</div>
<div id="online-mendelian-inheritance-in-man-omim"
class="section level2">
<h2>Online Mendelian Inheritance in Man (OMIM)</h2>
<p><br></p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-be7ef12aac2f96185af6" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-be7ef12aac2f96185af6">{"x":{"filter":"none","vertical":false,"data":[["BCL3","BIN1","RELB","EPHX2","APOE","APOE","APOE","APOE","APOE","APOE","APOE","APOC2","BLOC1S3","CEACAM16","CEACAM16","CR2","CR2","CR1","CR1","CR1","CR1","PICALM","CLCN1","CLCN1","CLCN1","CD2AP","TTR","TTR","TTR","DSG2","DSG2","SLC24A4","SLC24A4","SLC24A4"],["TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG","TG"],[1,1,1,0.9999999999976743,1,1,1,1,1,1,1,1,0.9999999999972153,0.999999999873957,0.999999999873957,0.9999992673703606,0.9999992673703606,0.9999992674241207,0.9999992674241207,0.9999992674241207,0.9999992674241207,0.9968214201251187,0.9881846535064829,0.9881846535064829,0.9881846535064829,0.7003741638158492,0.6378216689037693,0.6378216689037693,0.6378216689037693,0.6378279987931024,0.6378279987931024,0.09491209568051062,0.09491209568051062,0.09491209568051062],[1,1,1,1,1,1,1,1,1,1,1,1,0.9999999999999487,0.9999999999950631,0.9999999999950631,0.9999999014520241,0.9999999014520241,0.9999999015094846,0.9999999015094846,0.9999999015094846,0.9999999015094846,0.999651259191367,0.9979390751319739,0.9979390751319739,0.9979390751319739,0.9544074711743679,0.9211139741525487,0.9211139741525487,0.9211139741525487,0.9211517461508155,0.9211517461508155,0.4589597959861796,0.4589597959861796,0.4589597959861796],[1.000000000000001,1,0.9999999999999999,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999998,0.9999999999999997,0.9999999999999617,0.9999999999989159,0.9999999999989159,0.9999999750058211,0.9999999750058211,0.9999999409479006,0.9999999409479006,0.9999999409479006,0.9999999409479006,0.9998720754149846,0.9988159949683191,0.9988159949683191,0.9988159949683191,0.9780467935464852,0.9356918128525192,0.9356918128525192,0.9356918128525192,0.929205960903237,0.929205960903237,0.7134973578122618,0.7134973578122618,0.7134973578122618],["Leukemia/lymphoma, B-cell, 3 (2)","Centronuclear myopathy 2, 255200 (3)","?Immunodeficiency 53, 617585 (3)","{Hypercholesterolemia, familial, due to LDLR defect, modifier of}, 143890 (3)","Alzheimer disease 2, 104310 (3)","Hyperlipoproteinemia, type III, 617347 (3)","Lipoprotein glomerulopathy, 611771 (3)","Sea-blue histiocyte disease, 269600 (3)","{?Alzheimer disease, protection against, due to APOE3-Christchurch}, 607822 (3)","{?Macular degeneration, age-related}, 603075 (3)","{Coronary artery disease, severe, susceptibility to}, 617347 (3)","Hyperlipoproteinemia, type Ib, 207750 (3)","Hermansky-Pudlak syndrome 8, 614077 (3)","Deafness, autosomal dominant 4B, 614614 (3)","Deafness, autosomal recessive 113, 618410 (3)","Immunodeficiency, common variable, 7, 614699 (3)","{Systemic lupus erythematosus, susceptibility to, 9}, 610927 (3)","CR1 deficiency (1)","[Blood group, Knops system], 607486 (3)","{?SLE susceptibility} (1)","{Malaria, severe, resistance to}, 611162 (3)","Leukemia, acute myeloid, somatic, 601626 (3)","Myotonia congenita, dominant, 160800 (3)","Myotonia congenita, recessive, 255700 (3)","Myotonia levior, recessive (3)","Glomerulosclerosis, focal segmental, 3, 607832 (3)","Amyloidosis, hereditary, transthyretin-related, 105210 (3)","Carpal tunnel syndrome, familial, 115430 (3)","[Dystransthyretinemic hyperthyroxinemia], 145680 (3)","Arrhythmogenic right ventricular dysplasia 10, 610193 (3)","Cardiomyopathy, dilated, 1BB, 612877 (3)","Amelogenesis imperfecta, type IIA5, 615887 (3)","[Skin/hair/eye pigmentation 6, blond/brown hair], 210750 (3)","[Skin/hair/eye pigmentation 6, blue/green eyes], 210750 (3)"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Gene<\/th>\n      <th>Role<\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>base<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>near<\/sup><\/th>\n      <th><i>P<\/i><sub>1<\/sub><sup>bma<\/sup><\/th>\n      <th>Disease, OMIM ID<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4]},{"name":"Gene","targets":0},{"name":"Role","targets":1},{"name":"<i>P<\/i><sub>1<\/sub><sup>base<\/sup>","targets":2},{"name":"<i>P<\/i><sub>1<\/sub><sup>near<\/sup>","targets":3},{"name":"<i>P<\/i><sub>1<\/sub><sup>bma<\/sup>","targets":4},{"name":"Disease, OMIM ID","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>─ Session info ───────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.4.1 (2024-06-14)
 os       macOS Sonoma 14.6.1
 system   aarch64, darwin20
 ui       X11
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/Los_Angeles
 date     2024-09-17
 pandoc   3.1.11 @ /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/aarch64/ (via rmarkdown)

─ Packages ───────────────────────────────────────────────────────────────────
 package          * version    date (UTC) lib source
 AnnotationDbi      1.66.0     2024-06-16 [1] Bioconductor 3.19 (R 4.4.0)
 Biobase            2.64.0     2024-04-30 [1] Bioconductor 3.19 (R 4.4.0)
 BiocFileCache      2.12.0     2024-04-30 [1] Bioconductor 3.19 (R 4.4.0)
 BiocGenerics       0.50.0     2024-04-30 [1] Bioconductor 3.19 (R 4.4.0)
 biomaRt          * 2.60.1     2024-06-26 [1] Bioconductor 3.19 (R 4.4.0)
 Biostrings         2.72.1     2024-06-02 [1] Bioconductor 3.19 (R 4.4.0)
 bit                4.0.5      2022-11-15 [1] CRAN (R 4.4.0)
 bit64              4.0.5      2020-08-30 [1] CRAN (R 4.4.0)
 blob               1.2.4      2023-03-17 [1] CRAN (R 4.4.0)
 bslib              0.8.0      2024-07-29 [1] CRAN (R 4.4.0)
 cachem             1.1.0      2024-05-16 [1] CRAN (R 4.4.0)
 cli                3.6.3      2024-06-21 [1] CRAN (R 4.4.0)
 crayon             1.5.3      2024-06-20 [1] CRAN (R 4.4.0)
 crosstalk          1.2.1      2023-11-23 [1] CRAN (R 4.4.0)
 curl               5.2.2      2024-08-26 [1] CRAN (R 4.4.1)
 data.table       * 1.16.0     2024-08-27 [1] CRAN (R 4.4.1)
 DBI                1.2.3      2024-06-02 [1] CRAN (R 4.4.0)
 dbplyr             2.5.0      2024-03-19 [1] CRAN (R 4.4.0)
 devtools           2.4.5      2022-10-11 [1] CRAN (R 4.4.0)
 digest             0.6.37     2024-08-19 [1] CRAN (R 4.4.1)
 dplyr            * 1.1.4      2023-11-17 [1] CRAN (R 4.4.0)
 DT               * 0.33       2024-04-04 [1] CRAN (R 4.4.0)
 ellipsis           0.3.2      2021-04-29 [1] CRAN (R 4.4.0)
 evaluate           0.24.0     2024-06-10 [1] CRAN (R 4.4.0)
 fansi              1.0.6      2023-12-08 [1] CRAN (R 4.4.0)
 fastmap            1.2.0      2024-05-15 [1] CRAN (R 4.4.0)
 filelock           1.0.3      2023-12-11 [1] CRAN (R 4.4.0)
 fs                 1.6.4      2024-04-25 [1] CRAN (R 4.4.0)
 generics           0.1.3      2022-07-05 [1] CRAN (R 4.4.0)
 GenomeInfoDb       1.40.1     2024-06-16 [1] Bioconductor 3.19 (R 4.4.0)
 GenomeInfoDbData   1.2.12     2024-09-16 [1] Bioconductor
 git2r              0.33.0     2023-11-26 [1] CRAN (R 4.4.0)
 glue               1.7.0      2024-01-09 [1] CRAN (R 4.4.0)
 hms                1.1.3      2023-03-21 [1] CRAN (R 4.4.0)
 htmltools          0.5.8.1    2024-04-04 [1] CRAN (R 4.4.0)
 htmlwidgets        1.6.4      2023-12-06 [1] CRAN (R 4.4.0)
 httpuv             1.6.15     2024-03-26 [1] CRAN (R 4.4.0)
 httr               1.4.7      2023-08-15 [1] CRAN (R 4.4.0)
 httr2              1.0.4      2024-09-13 [1] CRAN (R 4.4.1)
 IRanges            2.38.1     2024-07-03 [1] Bioconductor 3.19 (R 4.4.1)
 jquerylib          0.1.4      2021-04-26 [1] CRAN (R 4.4.0)
 jsonlite           1.8.8      2023-12-04 [1] CRAN (R 4.4.0)
 KEGGREST           1.44.1     2024-06-19 [1] Bioconductor 3.19 (R 4.4.0)
 knitr              1.48       2024-07-07 [1] CRAN (R 4.4.0)
 later              1.3.2      2023-12-06 [1] CRAN (R 4.4.0)
 lifecycle          1.0.4      2023-11-07 [1] CRAN (R 4.4.0)
 magrittr           2.0.3      2022-03-30 [1] CRAN (R 4.4.0)
 memoise            2.0.1      2021-11-26 [1] CRAN (R 4.4.0)
 mime               0.12       2021-09-28 [1] CRAN (R 4.4.0)
 miniUI             0.1.1.1    2018-05-18 [1] CRAN (R 4.4.0)
 pillar             1.9.0      2023-03-22 [1] CRAN (R 4.4.0)
 pkgbuild           1.4.4      2024-03-17 [1] CRAN (R 4.4.0)
 pkgconfig          2.0.3      2019-09-22 [1] CRAN (R 4.4.0)
 pkgload            1.4.0      2024-06-28 [1] CRAN (R 4.4.0)
 png                0.1-8      2022-11-29 [1] CRAN (R 4.4.0)
 prettyunits        1.2.0      2023-09-24 [1] CRAN (R 4.4.0)
 profvis            0.3.8      2023-05-02 [1] CRAN (R 4.4.0)
 progress           1.2.3      2023-12-06 [1] CRAN (R 4.4.0)
 promises           1.3.0      2024-04-05 [1] CRAN (R 4.4.0)
 purrr              1.0.2      2023-08-10 [1] CRAN (R 4.4.0)
 R6                 2.5.1      2021-08-19 [1] CRAN (R 4.4.0)
 rappdirs           0.3.3      2021-01-31 [1] CRAN (R 4.4.0)
 Rcpp               1.0.13     2024-07-17 [1] CRAN (R 4.4.0)
 remotes            2.5.0      2024-03-17 [1] CRAN (R 4.4.0)
 rlang              1.1.4      2024-06-04 [1] CRAN (R 4.4.0)
 rmarkdown          2.28       2024-08-17 [1] CRAN (R 4.4.0)
 rprojroot          2.0.4      2023-11-05 [1] CRAN (R 4.4.0)
 RSQLite            2.3.7      2024-05-27 [1] CRAN (R 4.4.0)
 rstudioapi         0.16.0     2024-03-24 [1] CRAN (R 4.4.0)
 S4Vectors          0.42.1     2024-07-03 [1] Bioconductor 3.19 (R 4.4.1)
 sass               0.4.9.9000 2024-07-11 [1] Github (rstudio/sass@9228fcf)
 sessioninfo        1.2.2      2021-12-06 [1] CRAN (R 4.4.0)
 shiny              1.9.1      2024-08-01 [1] CRAN (R 4.4.0)
 stringi            1.8.4      2024-05-06 [1] CRAN (R 4.4.0)
 stringr            1.5.1      2023-11-14 [1] CRAN (R 4.4.0)
 tibble             3.2.1      2023-03-20 [1] CRAN (R 4.4.0)
 tidyr            * 1.3.1      2024-01-24 [1] CRAN (R 4.4.0)
 tidyselect         1.2.1      2024-03-11 [1] CRAN (R 4.4.0)
 UCSC.utils         1.0.0      2024-05-06 [1] Bioconductor 3.19 (R 4.4.0)
 urlchecker         1.0.1      2021-11-30 [1] CRAN (R 4.4.0)
 usethis            3.0.0      2024-07-29 [1] CRAN (R 4.4.0)
 utf8               1.2.4      2023-10-22 [1] CRAN (R 4.4.0)
 vctrs              0.6.5      2023-12-01 [1] CRAN (R 4.4.0)
 whisker            0.4.1      2022-12-05 [1] CRAN (R 4.4.0)
 withr              3.0.1      2024-07-31 [1] CRAN (R 4.4.0)
 workflowr          1.7.1      2023-08-23 [1] CRAN (R 4.4.0)
 xfun               0.47       2024-08-17 [1] CRAN (R 4.4.0)
 xml2               1.3.6      2023-12-04 [1] CRAN (R 4.4.0)
 xtable             1.8-4      2019-04-21 [1] CRAN (R 4.4.0)
 XVector            0.44.0     2024-04-30 [1] Bioconductor 3.19 (R 4.4.0)
 yaml               2.3.10     2024-07-26 [1] CRAN (R 4.4.0)
 zlibbioc           1.50.0     2024-04-30 [1] Bioconductor 3.19 (R 4.4.0)

 [1] /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/library

──────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
