---
title: "Simulated summary-level data based on WTCCC & Liver-specific network"
author: "Xiang Zhu"
date: "2018-09-19"
output: workflowr::wflow_html
---

```{r, echo=FALSE}
suppressPackageStartupMessages(library(R.matlab))
suppressPackageStartupMessages(library(ggpubr))
suppressPackageStartupMessages(library(plotROC))

data_path <- "~/Dropbox/rss/Data/peca_human/out_market/"
```

```{r, echo=FALSE}
viz_res <- function(pip,pve) {
  res <- R.matlab::readMat(paste0(data_path,"wtccc_liver_pip",100*pip,"pve",100*pve,"_seed644_out.mat"))
  
  cat("True total number of genes with nonzero effect: ", sum(c(res$gamma)!=0), "\n")
  cat("Estimated total number of genes with nonzero effect: ", sum(c(res$vb.weight[,2])), "\n")
  
  df <- data.frame(
    true_gamma <- c(res$gamma),
    estimate_gamma <- c(res$vb.mean[,2] * res$vb.weight[,2])
  )
  names(df) <- c("true_gamma","estimated_gamma")

  sp <- ggpubr::ggscatter(df, x="true_gamma", y="estimated_gamma", xlab="True gamma", ylab="Estimated gamma", cor.coef=TRUE, cor.coeff.args=list(method="pearson", label.x=-3, label.sep="\n"))
  sp <- sp + geom_abline(slope=1, intercept=0, color="red") + ggtitle(paste0("Estimate gene-level effects (all ",dim(df)[1]," genes)"))
  
  roc_df <- data.frame(D=ifelse(c(res$gamma)!=0, 1, 0), M=c(res$vb.weight[,2]))
  
  rp <- ggplot(roc_df, aes(d=D, m=M)) + geom_roc(n.cuts=0, color="blue") + style_roc() + theme_pubr() 
  rp <- rp + annotate("text", x=0.75, y=0.25, label=paste0("AUC = ", round(calc_auc(rp)$AUC, 3)))
  rp <- rp + geom_abline(intercept=0, slope=1, linetype="dashed") + ggtitle(paste0("Power to detect gene-level effects (all ",dim(df)[1]," genes)"))
  
  # viz network-specific gene-level results
  
  # load gene info
  gene_info <- readRDS("~/Dropbox/rss/Data/peca_human/data_factory/20180820/Homo_sapiens_GRCh37_gene_info.rds")
  
  # load network info
  full_net <- data.table::fread("~/Dropbox/rss/Data/peca_human/input_data/peca2/Liver_network.txt")

  # keep columns related to TG or TF only
  tgtf_df <- full_net[, c("TG","TF","Score")]
  tgtf_df <- unique(tgtf_df)
  names(tgtf_df) <- c("tg","tf","score")

  # remove rows where TG == TF
  tgtf_df <- tgtf_df[tgtf_df$tg != tgtf_df$tf, ]

  if(sum(tgtf_df$tg==tgtf_df$tf)>0) {
    stop("The current pipeline does not allow cases where TG==TF ...\n")
  }
  
  # locate TGs and TFs
  tg_nid <- gene_info$gene_nid[gene_info$hgnc_symbol %in% unique(tgtf_df$tg)]
  tf_nid <- gene_info$gene_nid[gene_info$hgnc_symbol %in% unique(tgtf_df$tf)]
  tgtf_nid <- unique(c(tg_nid,tf_nid))
  
  sp_tgtf <- ggpubr::ggscatter(df[tgtf_nid, ], x="true_gamma", y="estimated_gamma", xlab="True gamma", ylab="Estimated gamma", cor.coef=TRUE, cor.coeff.args=list(method="pearson", label.x=-3, label.sep="\n"))
  sp_tgtf <- sp_tgtf + geom_abline(slope=1, intercept=0, color="red") + ggtitle(paste0("Estimate gene-level effects (",length(tgtf_nid)," TGs and/or TFs)"))
  
  roc_df <- data.frame(D=ifelse(c(res$gamma[tgtf_nid])!=0, 1, 0), M=c(res$vb.weight[tgtf_nid,2]))
  
  rp_tgtf <- ggplot(roc_df, aes(d=D, m=M)) + geom_roc(n.cuts=0, color="blue") + style_roc() + theme_pubr() 
  rp_tgtf <- rp_tgtf + annotate("text", x=0.75, y=0.25, label=paste0("AUC = ", round(calc_auc(rp_tgtf)$AUC, 3)))
  rp_tgtf <- rp_tgtf + geom_abline(intercept=0, slope=1, linetype="dashed") + ggtitle(paste0("Power to detect gene-level effects (",length(tgtf_nid)," TGs and/or TFs)"))
  
  sp_tg <- ggpubr::ggscatter(df[tg_nid, ], x="true_gamma", y="estimated_gamma", xlab="True gamma", ylab="Estimated gamma", cor.coef=TRUE, cor.coeff.args=list(method="pearson", label.x=-3, label.sep="\n"))
  sp_tg <- sp_tg + geom_abline(slope=1, intercept=0, color="red") + ggtitle(paste0("Estimate gene-level effects (",length(tg_nid)," TGs)"))
  
  roc_df <- data.frame(D=ifelse(c(res$gamma[tg_nid])!=0, 1, 0), M=c(res$vb.weight[tg_nid,2]))
  
  rp_tg <- ggplot(roc_df, aes(d=D, m=M)) + geom_roc(n.cuts=0, color="blue") + style_roc() + theme_pubr() 
  rp_tg <- rp_tg + annotate("text", x=0.75, y=0.25, label=paste0("AUC = ", round(calc_auc(rp_tg)$AUC, 3)))
  rp_tg <- rp_tg + geom_abline(intercept=0, slope=1, linetype="dashed") + ggtitle(paste0("Power to detect gene-level effects (",length(tg_nid)," TGs)"))
  
  sp_tf <- ggpubr::ggscatter(df[tf_nid, ], x="true_gamma", y="estimated_gamma", xlab="True gamma", ylab="Estimated gamma", cor.coef=TRUE, cor.coeff.args=list(method="pearson", label.x=-3, label.sep="\n"))
  sp_tf <- sp_tf + geom_abline(slope=1, intercept=0, color="red") + ggtitle(paste0("Estimate gene-level effects (",length(tf_nid)," TFs)"))
  
  roc_df <- data.frame(D=ifelse(c(res$gamma[tf_nid])!=0, 1, 0), M=c(res$vb.weight[tf_nid,2]))
  
  rp_tf <- ggplot(roc_df, aes(d=D, m=M)) + geom_roc(n.cuts=0, color="blue") + style_roc() + theme_pubr() 
  rp_tf <- rp_tf + annotate("text", x=0.75, y=0.25, label=paste0("AUC = ", round(calc_auc(rp_tf)$AUC, 3)))
  rp_tf <- rp_tf + geom_abline(intercept=0, slope=1, linetype="dashed") + ggtitle(paste0("Power to detect gene-level effects (",length(tf_nid)," TFs)"))
  
  # viz whole-genome SNP-level results
  df_snp <- data.frame(
    true_beta <- c(res$beta),
    estimate_beta <- c(res$beta.est)
  )
  names(df_snp) <- c("true_beta","estimated_beta")

  sp_snp <- ggpubr::ggscatter(df_snp, x="true_beta", y="estimated_beta", xlab="True beta", ylab="Estimated beta", cor.coef=TRUE, cor.coeff.args=list(method="pearson", label.x=-10, label.sep="\n"))
  sp_snp <- sp_snp + geom_abline(slope=1, intercept=0, color="red") + ggtitle(paste0("Estimate SNP-level effects (all ",dim(df_snp)[1]," SNPs)"))
  
  roc_df_snp <- data.frame(D=ifelse(abs(res$beta)>=1, 1, 0), M=c(res$alpha.snp))
  
  rp_snp <- ggplot(roc_df_snp, aes(d=D, m=M)) + geom_roc(n.cuts=0, color="blue") + style_roc() + theme_pubr() 
  rp_snp <- rp_snp + annotate("text", x=0.75, y=0.25, label=paste0("AUC = ", round(calc_auc(rp_snp)$AUC, 3)))
  rp_snp <- rp_snp + geom_abline(intercept=0, slope=1, linetype="dashed") + ggtitle(paste0("Power to detect SNP-level effects (all ",dim(df_snp)[1]," SNPs)"))

  print(ggarrange(sp, rp, sp_tgtf, rp_tgtf, sp_tg, rp_tg, sp_tf, rp_tf, sp_snp, rp_snp, ncol=2, nrow=5))
}
```

## PIP: `r pip=0.1; pip` & PVE: `r pve=0.2; pve`

```{r, fig.width=12, fig.height=30, echo=FALSE}
viz_res(pip,pve)
```

## PIP: `r pip=0.2; pip` & PVE: `r pve=0.2; pve`

```{r, fig.width=12, fig.height=30, echo=FALSE}
viz_res(pip,pve)
```

## PIP: `r pip=0.2; pip` & PVE: `r pve=0.4; pve`

```{r, fig.width=12, fig.height=30, echo=FALSE}
viz_res(pip,pve)
```
